docs/core/deploying/trim-warnings/il2032.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "IL2032: Unrecognized value passed to the parameter 'parameter' of 'System.Activator.CreateInstance' method"
+description: "Learn about trim warning IL2032: CreateInstanceUnrecognizedParameter"
+ms.date: 08/25/2021
+ms.topic: reference
+author: mateoatr
+ms.author: matorre
+f1_keywords:
+  - "IL2032"
+---
+# IL2032: Unrecognized value passed to the parameter 'parameter' of 'System.Activator.CreateInstance' method
+
+## Cause
+
+The value passed to the assembly or type name of the <xref:System.Activator.CreateInstance>
+method cannot be statically analyzed. The Trimmer cannot guarantee the availability of
```

Comment:
Why are we capitalizing the "Trimmer" - maybe there's a grammatical reason, but just looking at it, it looks weird - I would simply say "the trimmer". Similarly to "the compiler" (I've never seen that capitalized either).